api/gen/http/openapi.yaml

```diff
@@ -13,28 +13,137 @@ produces:
       schemes:
       - http
-  /resources:
+  /swagger/swagger.json:
     get:
       tags:
-      - api
-      summary: list api
-      description: Get all tasks and pipelines.
-      operationId: api#list
+      - swagger
+      summary: Download gen/http/openapi.json
+      description: JSON document containing the API swagger definition
+      operationId: swagger#/swagger/swagger.json
       responses:
         "200":
-          description: OK response.
+          description: File downloaded
+          schema:
+            type: file
       schemes:
       - http
+definitions:
+  CategoryAllInternalErrorResponseBody:
+    title: 'Mediatype identifier: application/vnd.goa.error; view=default'
+    type: object
+    properties:
+      fault:
+        type: boolean
+        description: Is the error a server-side fault?
+        example: true
+      id:
+        type: string
+        description: ID is a unique identifier for this particular occurrence of the
+          problem.
+        example: 123abc
+      message:
+        type: string
+        description: Message is a human-readable explanation specific to this occurrence
+          of the problem.
+        example: parameter 'p' must be an integer
+      name:
+        type: string
+        description: Name is the name of this class of errors.
+        example: bad_request
+      temporary:
+        type: boolean
+        description: Is the error temporary?
+        example: true
+      timeout:
+        type: boolean
+        description: Is the error a timeout?
+        example: false
+    description: All_internal-error_Response_Body result type (default view)
+    example:
+      fault: true
+      id: 123abc
+      message: parameter 'p' must be an integer
+      name: bad_request
+      temporary: false
+      timeout: true
+    required:
+    - name
+    - id
+    - message
+    - temporary
+    - timeout
+    - fault
+  TagResponse:
+    title: TagResponse
+    type: object
+    properties:
+      id:
+        type: integer
+        description: Id is the unique id of tags
+        example: 1
+        format: int64
+      name:
+        type: string
+        description: name of tag
+        example: cli
+    example:
+      id: 1
+      name: cli
+    required:
+    - id
+    - name
+  categoryResponse:
+    title: categoryResponse
+    type: object
+    properties:
+      id:
+        type: integer
+        description: unique id of category
+        example: 1
+        format: int64
+      name:
+        type: string
+        description: name of category
+        example: Image-build
+      tags:
+        type: array
+        items:
+          $ref: '#/definitions/TagResponse'
+        description: list of tag associated with category
+        example:
+        - id: 1
+          name: cli
+        - id: 1
+          name: cli
+    example:
+      id: 1
+      name: Image-build
+      tags:
+      - id: 1
+        name: cli
+      - id: 1
```

Comment:
the id repeats, is that on purpose?

Reply:
no,it repeats for Attribute having type ArrayOf()